frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -289,3 +329,42 @@ function buildConfusionMatrixConfig(
     },
   ];
 }
+
+interface ScalarMetricsSectionProps {
+  artifact: Artifact;
+}
+function ScalarMetricsSection({ artifact }: ScalarMetricsSectionProps) {
+  const customProperties = artifact.getCustomPropertiesMap();
+  const name = customProperties.get('name')?.getStringValue();
+  const data = customProperties
+    .getEntryList()
+    .map(x => ({
+      key: x[0],
+      value: JSON.stringify(getMetadataValue(customProperties.get(x[0]))),
+    }))
```

Comment:
nit: you can deconstruct an array right from the arg list

```suggestion
    .map(([key]) => ({
      key,
      value: JSON.stringify(getMetadataValue(customProperties.get(key))),
    }))
```

Reply:
I see, adapting the change.